src/shopping-list/shopping-list.css

```diff
@@ -0,0 +1,170 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+* {
+  font-family: 'Roboto Condensed', sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 1200px;
+  margin: auto;
+  background-color: #f5f5f5;
+  color: #4e4e4e;
+}
+
+h3 {
+  margin: 0;
+}
+
+h5 {
+  padding-left: 10px;
+  margin: 1rem 0 0.6rem;
+}
+
+main {
+  margin: 4rem 2rem;
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  grid-gap: 8rem;
+  padding: 2rem;
+}
+
+.list-ingredients__container {
+  padding: 0.3rem 1.3rem;
+  margin-bottom: 0.3rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 16px;
+  word-break: break-all;
+  cursor: default;
+}
+
+.list-ingredients__container--add-button {
+  padding: 0.17rem 0.55rem;
+  margin-left: 12px;
+  border: none;
+  border-radius: 5px;
+  background: rgb(255, 255, 255);
+  color: rgb(143, 143, 143);
+  box-shadow: 0.3px 0.3px 3px #a0a0a0, -0.3px -0.3px 4px #f8f8f8;
+  font-weight: bold;
+  cursor: pointer;
+  font-size: 14px;
+}
+
+.list-recipies__container {
+  max-width: 16rem;
+  border: 1px solid #fff;
+  border-radius: 0 0 7px 7px;
+  background-color: rgb(252, 252, 252);
+  margin-top: -5px;
+  box-shadow: 6.91px 6.91px 20px #d6d6d6, -6.91px -6.91px 20px #ffffff;
+  font-size: 22px;
+  cursor: pointer;
+}
+
+.list-recipies__container--element {
+}
+
+.list-recipies__container--element::after {
+  content: '';
+  display: block;
+  width: 92%;
+  margin: 1rem auto;
+  height: 1px;
+  background-color: rgb(206, 206, 206);
+}
+
+.list-recipies__container--element:last-of-type::after {
+  content: '';
+  display: none;
+}
+
+.shopping-list {
+  margin-top: 5rem;
+  min-width: 32rem;
+  min-height: 15rem;
+  padding: 2rem 3rem;
+  border-radius: 16px;
+  background: #f5f5f5;
+  box-shadow: 24.62px 24.62px 20px #cecece, -24.62px -24.62px 20px #ffffff;
+}
+
+.shopping-list__header {
+  font-size: 28px;
+  text-align: center;
+}
+
+.shopping-list__ingredients-container {
+  font-size: 16px;
+  font-weight: 400;
+  margin-top: 1.5rem;
```

Comment:
you can use * + * selector to get one global margin-top

Reply:
I don't feel like I have components that have the same styling, but I will double-check that. Thanks for the suggestion, I do appreciate!